src/app/modules/account/login/login.component.scss

```diff
@@ -46,6 +45,10 @@
   .institutions {
     width: 100%;
     display: flex;
+    flex-wrap: wrap;
+    h4{
+      white-space: nowrap;
+    }
```

Comment:
Not really related to the issue, but because you already worked on it....

![afbeelding](https://user-images.githubusercontent.com/55654104/94115330-dd793f80-fe49-11ea-8494-57aed5b2b50d.png)

Could you take a look at this? The screen width is 463px